src/components/Sticky/Sticky.tsx

```diff
@@ -106,3 +107,7 @@ export default class Sticky extends React.Component<Props, State> {
     }
   }
 }
+
+function isFunction(arg: any): arg is Function {
```

Comment:
Should this be available as a utility somewhere? Or… should we use Lodash's isFunction instead?

Reply:
It's so small I don't think it's worth it. It's a bit of a hack to work around some of typescript's type shortening in this one case rather than something generally useful.

Reply:
We need this because of the 'arg is function' typescript hint. using lodash's isFunction wouldn't give us that.